docs/context/development/schema.json

```diff
@@ -332,6 +339,25 @@
             "items": {
               "$ref": "#/definitions/taxonomic_unit"
             }
+          },
+          "apomorphy": {
+            "type": "object",
+            "description": "For a phyloreference using an apomorphy, this field describes the character being used as the apomorphy",
+            "additionalProperties": true,
+            "properties": {
+              "definition": {
+                "description": "A description of this apomorphy."
+              },
+              "phenotypicQuality": {
+                "description": "The quality of this phenotype. Must be a subclass of pato:quality."
+              },
+              "phenotypeBearer": {
+                "description": "The phenotype being used as an apomorphy."
```

Comment:
No, this is not the phenotype, it's the bearer of the phenotype. 😞

Reply:
So, this should be "The bearer of the phenotype being used as an apomorphy."? I'm having a hard time wrapping my head around the idea that an UBERON term (such as "[turtle shell](http://purl.obolibrary.org/obo/UBERON_0008271)") is the bearer of the phenotype, whereas I would intuitively think of the bearer of the phenotype as an individual turtle with such as shell. However, if that is the right phrasing from an UBERON/PATO perspective (i.e. if that's what people who think in terms of phenotypeBearer/phenotypicQuality would understand), then that is absolutely what we should use.

Reply:
The individual organism exhibits the phenotype. The turtle shell is the bearer of the phenotypic quality ("present") (and to be correct, not of the phenotype, which is the combination of quality and bearing entity; I corrected that).

Reply:
So `phenotypeBearer` is therefore not quite correct, you are right. The entity is the bearer of the phenotypic quality, not the phenotype. Maybe use `bearingEntity` instead?

Reply:
Sounds good! I've renamed that property to `bearingEntity` in fb09cc0263a.